Week1/homework/app.js

```diff
@@ -1,11 +1,163 @@
+ "de_man_van_de_bloemen" : {adress: "https://s.s-bol.com/imgbase0/imagebase3/large/FC/6/7/6/1/9200000077291676.jpg"},
+ "ren_amina_ren" : {adress: "https://s.s-bol.com/imgbase0/imagebase3/large/FC/4/2/9/9/9200000056169924.jpg"},
+ "het_diner" : {adress: "https://www.lezenvooriedereen.be/media/catalog/product/cache/2/image/9df78eab33525d08d6e5fb8d27136e95/d/i/diner_lowres.jpg"},
+ };
+ // step 2.5
 
+ function makeList() {
+  let bookcontainer = document.getElementById("mybooks");
+  let div = document.createElement("div");
+  bookcontainer.appendChild(div);
+  
+
+  for (let i = 0; i<bookTitles.length; i++) {
+    
+    let contentTitle = document.createTextNode(bookDetails[bookTitles[i]].title);
```

Comment:
I saw you are using `bookDetails[bookTitles[i]]` in a multiple places with this long format.
it could more readable if you defined it in a variable at the beginning of your for loop and using it everywhere else.